### ISSUE TITLE
Missing alt text in posts received from Mastodon and Pleroma

### ISSUE THREAD

**Issue by GidiKroon** [pixelfed/pixelfed#2463]
When a post is made on Mastodon or Pleroma which contains a photo, it is shown in the Pixelfed timeline. If the photo has descriptive text (alt text) set, the alt text in Pixelfed still says "Photo was not tagged with any alt text". It would be great if incoming alt text was received and presented. Outgoing alt text works fine.

Tested with pixelfed.social, mastodon.social and my pleroma instance which is at the latest stable release (2.2.0).

**Comment by dansup** [comment]
Fixed!
